R/methods_BayesFactor.R

```diff
@@ -153,6 +153,11 @@ model_parameters.BFBayesFactor <- function(model,
   attr(out, "pretty_names") <- pretty_names
   class(out) <- c("parameters_model", "see_parameters_model", class(out))
 
+  # converting CI to fraction
+  if ("CI" %in% names(out)) {
+    out$CI <- out$CI / 100
+  }
+
```

Comment:
Here you have to coordinate with @DominiqueMakowski and _bayestestR_, else we just have 1% of the CI. Thus, you need to remove this code again and do all the CI-fraction-stuff in `bayestestR::describe_posterior()`.